### DISCUSSION TITLE
Downtime from the 13/14th of March 2024

### DISCUSSION THREAD

**Discussion by cyrneko** [stoatchat/discussions#680] [General]
Apparently, Revolt is currently inaccessible in what I speculate to be a hilarious yet misfortunate situation in which someone has made Revolt (Web and Desktop) point to http://local.revolt.chat:8000 instead of the main production servers, causing:

- A "Third-party instance" warning in official web/desktop client(s)
- Errors in the console
- The inability to login, register, or do anything else that requires access to the Revolt API.

I don't know jackshit about the development and/or publishing flow, but because local.revolt.chat is supposed to be, well, *local development versions of Revolt*, it makes sense for now

**Comment by cyrneko** [comment]
![image](https://github.com/revoltchat/revolt/assets/58338843/c69fbbca-f023-4e9e-859e-70f93c028afc)

:arrow_up: Uploading this here for reference

**Comment by amycatgirl** [comment]
Also having the same issue here.

Downtime started almost an hour ago which was around the time commits c25ecc and 60fdb5c were pushed to the revoltchat/revite repo, both regarding the publishing of how deployment was handled.

I will assume that the issue is from those commits, or from some internal changes to the publishing script.

Honestly, I have no idea :woman_shrugging:

**Reply by insertish** [reply]
Had a rogue `.env.local` during the build, re-deployed with the correct variables.
This would've been avoided if the build worked in the first place on the server itself (dependencies were just not installing, not sure why), so I ended up off-loading it to another server which had the incorrect env setup.
